apps/communications/contacts/js/utilities/telemetry.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+/* global LazyLoader */
+/* global AdvancedTelemetryHelper */
+
+(function(exports) {
+
+  var namespace = 'telemetry_gaia_contacts';
+
+  var dependencies = LazyLoader.load(
```

Comment:
These dependecies are loaded when `js/utilities/telemetry.js` is loaded. Can we delay the load until `logImportUsage` is called?